eng/pipeline/stages/run-codeql.yml

```diff
@@ -118,6 +118,11 @@ stages:
                 chmod 755 "${WORKAROUND_DIR}/go"
                 export PATH="${WORKAROUND_DIR}:${PATH}"
 
+                # Set GOEXPERIMENT for linux arm32/arm64 to avoid crypto backend issues
+                if [ "${{ c.os }}" = "linux" ] && ([ "${{ c.arch }}" = "arm" ] || [ "${{ c.arch }}" = "arm64" ]); then
+                  export GOEXPERIMENT=nosystemcrypto
```

Comment:
How about setting it for all, for now, to reach parity?

A != check rather than = seems like the way to go, otherwise.

Reply:
agreed, I've updates the job to set `GOEXPERIMENT=nosystemcrypto` for all platforms now